cylc/flow/network/schema.py

```diff
@@ -1998,17 +1998,20 @@ class Arguments:
 class FlowMutationArguments:
     flow = graphene.List(
         graphene.NonNull(Flow),
-        default_value=[FLOW_ALL],
+        default_value=[],
         description=sstrip(f'''
-            The flow(s) to trigger these tasks in.
-
-            This should be a list of flow numbers OR a single-item list
-            containing one of the following three strings:
-
-            * {FLOW_ALL} - Triggered tasks belong to all active flows
-              (default).
-            * {FLOW_NEW} - Triggered tasks are assigned to a new flow.
-            * {FLOW_NONE} - Triggered tasks do not belong to any flow.
+            The flow(s) to trigger/set these tasks in.
+
+            By default:
+            * active tasks (n=0) keep their existing flow assignment
+            * inactive tasks (n>0) get assigned all active flows
+
+            Otherwise you can assign (inactive tasks) or add to (active tasks):
+            * a list of integer flow numbers
+            or a single-item list containing one of the following strings:
```

Comment:
```suggestion
            or one of the following strings:
```

Reply:
I think it is a list?

![Screenshot from 2024-10-14 11-15-30](https://github.com/user-attachments/assets/46d957e9-aebd-4b0e-8cf1-5321af34b50a)

Reply:
I think "one of the following strings" is clear enough, and won't give the impression you're supposed to type `[all]`